test/runtests.jl

```diff
@@ -24,6 +26,15 @@ using Impute:
     interp,
     chain
 
+function add_missings(X, ratio=0.1)
```

Comment:
I think I'd like this kind of thing to become an exported test suite, so folks can easily generate test cases on the fly. I can sort that out later though.

Reply:
We can separate tests to files and let some tests include the file that include this util function, but this job doesn't need to be done this PR.

Reply:
Would you mind making an issue then? Especially, since this is just copying what the svd PR is doing...

Reply:
If you merge SVD first, then I deal with conflict